.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "endOfLine": "lf",
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "es5"
```

Comment:
I don't like the trailing commas being added, e.g. after `end` and `}` in this example:
absoluteRange: {
               start,
               end,
  },
```

```

Reply:
Setting `trailingComma` to `none` should fix it.